File: python/8 kyu/243.py
# https://www.codewars.com/kata/5dd462a573ee6d0014ce715b/train/python

# Passed

def same_case(a, b): 
    if a.isalpha() and b.isalpha():
        # both are letters
        if (a.isupper() and b.isupper()) or (a.islower() and b.islower()):
            return 1 # same case
        else:
            return 0 # not same case
    
    return -1

output = same_case('C', 'B')
print(output)

File: nodejs/8 kyu/243.js
// https://www.codewars.com/kata/57ab2d6072292dbf7c000039/train/javascript

// Passed

function correctPolishLetters(string) {
  let polishToEnglishMap = {
    ą: "a",
    ć: "c",
    ę: "e",
    ł: "l",
    ń: "n",
    ó: "o",
    ś: "s",
    ź: "z",
    ż: "z",
  };

  let result = "";
  for (const letter of string) {
    if (letter in polishToEnglishMap) {
      result += polishToEnglishMap[letter];
    } else {
      result += letter;
    }
  }

  return result;
}

const output = correctPolishLetters("Jędrzej Błądziński");
console.log(output);